src/OrbitGl/CoreMath.h

```diff
@@ -24,4 +24,17 @@ using Color = gte::Vector4<unsigned char>;
 [[nodiscard]] inline Vec2 Vec3ToVec2(const Vec3& v) { return {v[0], v[1]}; }
 [[nodiscard]] inline Vec3 Vec2ToVec3(Vec2 vertex, float z = 0) { return {vertex[0], vertex[1], z}; }
 
+[[nodiscard]] inline bool IsInBetween(float point, float min, float max) {
```

Comment:
Agree with value, suggestion about naming?

Reply:
Not really.. I thought maybe there are good names from other programming languages/frameworks, but I didn't really find anything. From a maths dircection, maybe `IsInInterval` makes sense.. or `IsInBounds`. I dont have a strong opinion and `IsInBetween` is also fine

Reply:
These functions always have the problem that they don't properly define whether they operate on open, half-open or closed intervals. That's one of the big reasons you won't find them in standard libraries.

The proper solution would be to define a type: `ClosedInterval` and define the function as `bool IsElementOf(float value, ClosedInterval interval)`. In this way it can be overloaded later on for other types - like `OpenInterval` or whatever can fulfills the `IsElementOf` operation.

Reply:
What do you think now?